awx/api/serializers.py

```diff
@@ -2181,8 +2181,8 @@ def validate_credential_type(self, credential_type):
                 'insights_inventories',
                 'inventorysources',
                 'inventoryupdates',
-                'jobs',
-                'jobtemplates',
+                'unifiedjobs',
```

Comment:
This relation name changed because the M2M credentials relation moved to UJ and UJT.